scss/fonts.scss

```diff
@@ -1,4 +1,11 @@
+// Used for Buttons text and h3 header
```

Comment:
This is also used in navigation links right?

Reply:
Yeah. I think buttons cover both.